client/src/Components/Robot.js

```diff
@@ -41,24 +54,23 @@ class Robot extends Component {
   }
 
   render() {
-    // TODO: change play button depending on whether melody is currently playing.
     return (
       <div className="robot">
         <div className="robot-toolbar">
           <MDBIcon
             far
             size="lg"
-            icon="play-circle"
+            icon={this.props.isPlaying ? "pause-circle" : "play-circle"}
```

Comment:
It doesn't pause the playback, it'll completely stop it. Maybe use "stop-circle" instead?